src/site/_filters/gitlocalize-link.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require("path");
+const site = require("../_data/site");
+
+module.exports = (inputPath, lang) => {
+  if (!inputPath) {
+    /* eslint-disable-next-line */
+    console.warn(
+      "inputPath passed to gitlocalizeLink filter was undefined or null.",
+    );
+  }
+  inputPath = inputPath.replace(`/${lang}/`, `/${site.defaultLocale}/`);
```

Comment:
Can we sanity-check by adding e.g.

```js
if (!inputPath || inputPath.startsWith(`/${lang}/`)) {
  return inputPath;  // not a translated link, return early for safety
}
```

Reply:
I moved whole locale code to a separate module, and use it for sanity check here.